### DISCUSSION TITLE
Offline `pnpm deploy` and `pnpm dlx`

### DISCUSSION THREAD

**Discussion by ddadaal** [pnpm/pnpm#5761] [Ideas]
Is there an `--offline` option to prevent network connection when running `pnpm deploy` and `pnpm dlx`? It is strange that `pnpm deploy` or `pnpm dlx` still connects to registry even if I have already run `pnpm fetch` to fetch all packages into the registry.

![image](https://user-images.githubusercontent.com/8363856/206103636-19fca832-15e0-4c2f-a02a-805739e8966b.png)

**Comment by niklaskorz** [comment]
I'm running into the same issue, this makes it impossible to use in environments without internet connection
